src/uopool/services/uopool.rs

```diff
@@ -471,6 +490,71 @@ where
         }
     }
 
+    async fn handle_past_events(
+        &self,
+        request: tonic::Request<HandlePastEventRequest>,
+    ) -> Result<Response<HandlePastEventResponse>, tonic::Status> {
+        let req = request.into_inner();
+        let HandlePastEventRequest {
+            entry_point: entry_point_opt,
+            chain_id,
+        } = req;
+        let lastest_block = self
+            .eth_provider
+            .clone()
+            .get_block_number()
+            .await
+            .map_err(|e| {
+                tonic::Status::internal(format!("Getting the latest block number error: {e:?}"))
+            })?;
+        let last_block = std::cmp::max(
+            1u64,
+            lastest_block
+                .checked_sub(U64::from(1000))
```

Comment:
maybe make this value const and move to mod - so we change more easily change it later if needed.

Reply:
1000